core/src/main/java/org/elasticsearch/index/translog/TruncateTranslogCommand.java

```diff
@@ -168,7 +168,8 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
 
     /** Write a checkpoint file to the given location with the given generation */
     public static void writeEmptyCheckpoint(Path filename, int translogLength, long translogGeneration) throws IOException {
-        Checkpoint emptyCheckpoint = new Checkpoint(translogLength, 0, translogGeneration, SequenceNumbersService.UNASSIGNED_SEQ_NO);
+        Checkpoint emptyCheckpoint =
+            new Checkpoint(translogLength, 0, translogGeneration, Long.MAX_VALUE, Long.MIN_VALUE, SequenceNumbersService.UNASSIGNED_SEQ_NO);
```

Comment:
I pushed e6b356d885d22524bc899987525d6e655fffd31a although I left unsettled the issue of the default values for the min and max sequence number (I will address these later).